Resources/Private/Partials/Product/CartForm.html

```diff
@@ -11,7 +11,7 @@
             controller="Cart\Product"
             action="add"
             method="post"
-            pageType="{f:if(condition:'{settings.addToCartByAjax}', then:'{settings.addToCartByAjax}', else:'')}"
```

Comment:
This also leads to an error, because pageType must not be empty! You need to define 0 as fallback here.